bamazonSupervisor.js

```diff
@@ -0,0 +1,113 @@
+var mysql = require("mysql");
+
+var inquirer = require("inquirer");
+
+require("console.table");
+
+
+var connection = mysql.createConnection({
+  host: "localhost",
+  port: 3306,
+
+  // Your username
+  user: "root",
+
+  // Your password
+  password: "",
+  database: "bamazon"
+});
```

Comment:
It could make for a better experience for the user to include a ```process.env``` variable that gets assigned in the command line so that the user doesn't have to alter the file to make it work for their sql setup.